warden/x/warden/keeper/sign_requests.go

```diff
@@ -0,0 +1,56 @@
+package keeper
+
+import (
+	"cosmossdk.io/errors"
+	"crypto/ed25519"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	types "github.com/warden-protocol/wardenprotocol/warden/x/warden/types/v1beta3"
+)
+
+func (k Keeper) fullfilSignRequest(msg *types.MsgFulfilSignRequest, key types.Key, req types.SignRequest, ctx sdk.Context) error {
```

Comment:
let's follow the convention of having ctx as the first param

```suggestion
func (k Keeper) fullfilSignRequest(ctx sdk.Context, msg *types.MsgFulfilSignRequest, key types.Key, req types.SignRequest) error {
```